.changeset/fifty-buses-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+"@salt-ds/lab": patch
+---
+
+Drawer
+
+- Added `Drawer` component
```

Comment:
we've been adding a code block with default usage examples to the changeset, you might need one too